Client/src/components/contanier/staff/Student/AddStudent/index.css

```diff
@@ -9,41 +8,31 @@
   color: #06468F;
 }
 
+.add-student-title{
+  width: fit-content;
```

Comment:
@Fatmasiam I think you also need `width: -webkit-fit-content` to cover safari, underneath this.  And IE / Edge don't support fit-content 😂
Maybe there's an alternative way to style this title to avoid fit-content, like turning it into an inline-block or flex or something.